css/styles.css

```
@@ -831,56 +830,141 @@ body {
   color: inherit;
 }
 
+.catalog-header {
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: flex-start;
+  padding-top: 32px;
+  margin-bottom: 40px;
+}
+
+.catalog-title {
+  margin: 0;
+  padding: 0;
+  font-weight: 900;
+  font-size: 32px;
+  line-height: 46px;
+  color: #2d3440;
+}
+
 .breadcrumbs {
+  display: flex;
+  margin: 0;
+  margin-bottom: 16px;
+  padding: 0;
   list-style: none;
 }
 
+.breadcrumbs-item {
+  margin-right: 8px;
+}
+
+.breadcrumbs-item + .breadcrumbs-item::before {
+  content: "";
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  margin-right: 2px;
+  vertical-align: middle;
+  background-image: url("../img/chevrons-right.svg");
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+}
+
 .breadcrumbs-link {
+  text-decoration: none;
   font-weight: 700;
   font-size: 14px;
   line-height: 20px;
   color: #2d3440;
+  border-bottom: 1px solid #2d3440;
 }
 
-.catalog-title {
-  font-weight: 900;
-  font-size: 32px;
-  line-height: 46px;
-  color: #2d3440;
+.breadcrumbs-item:last-child .breadcrumbs-link{
```

Следи за исполнением кодгайдов:

```suggestion
.breadcrumbs-item:last-child .breadcrumbs-link {
```